ui/src/VoxettaUserForm.js

```diff
@@ -0,0 +1,215 @@
+/*
+Copyright 2020 Google LLC
+ 
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+ 
+    https://www.apache.org/licenses/LICENSE-2.0
+ 
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+ 
+import {LitElement, html, css} from 'lit-element';
+
+import {CookieService} from './CookieService';
+import {TextField} from '@material/mwc-textfield';
+import {Select} from '@material/mwc-select';
+import {ListItem} from '@material/mwc-list/mwc-list-item';
+import {Button} from '@material/mwc-button';
+ 
+/**
+ * Component responsible for providing users a means to provide
+ * their personal information. 
+ */
+export class VoxettaUserForm extends LitElement {
+
+    static get styles() {
+        return css`
+            .container {
+                align-items: center; 
+                display: flex; 
+                flex-direction: column; 
+                flex-wrap: wrap;
+                justify-content: center; 
+                text-align: center;  
+                width: 100%; 
+            }
+            h1.title {
+                font-family: 'Roboto';
+                font-size: 30px;
+            }
+            h2.description {
+                font-family: 'Roboto';
+                font-size: 20px;
+            }
+            mwc-textfield {
+                --mdc-theme-primary: #1a73e8;
+                padding: 10px; 
+                width: 300px;
+            }
+            mwc-select {
+                --mdc-theme-primary: #1a73e8;
+                padding: 10px; 
+                widith: 300px;
+            }
+            mwc-button {
+                padding: 10px;
+                width: 300px; 
+            }
+            mwc-button.save {
+                --mdc-theme-primary: #1a73e8;
+                --mdc-theme-on-primary: white;
+            }
+            mwc-button.cancel {
+                --mdc-theme-primary: white;
+                --mdc-theme-on-primary: #1a73e8;
+            }
+        `;
+    }
+
+    constructor() {
+        super();
+        this.cookieService = new CookieService();
+        this.userId = this.cookieService.getCookieValue("userId");
+        this.gender = this.cookieService.getCookieValue("gender");
+        this.userAge = this.cookieService.getCookieValue("userAge");
+        this.deviceType = this.cookieService.getCookieValue("deviceType"); 
```

Comment:
why is this necessary? I thought originally we don't show the form at all if a session cookie exists?or will we just always display it for now?

Reply:
This is for auto-populating when the user accesses the form via the user icon button on the record page in case they want to see their information/make any changes--right now there is no functionality for displaying the form from the beginning/skipping if it is already filled--will complete in a later pr